pkg/migrations/mls/20240109001927_add-messages.up.sql

```diff
@@ -0,0 +1,37 @@
+SET statement_timeout = 0;
+
+--bun:split
+
+CREATE TABLE group_messages (
+    id BIGSERIAL PRIMARY KEY,
+    created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    group_id BYTEA NOT NULL,
+    data BYTEA NOT NULL,
+    group_id_data_hash BYTEA NOT NULL
+);
+
+--bun:split
+
+CREATE INDEX idx_group_messages_group_id_created_at ON group_messages(group_id, created_at);
+
+--bun:split
+
+CREATE UNIQUE INDEX idx_group_messages_group_id_data_hash ON group_messages (group_id_data_hash);
+
+--bun:split
+
+CREATE TABLE welcome_messages (
+    id BIGSERIAL PRIMARY KEY,
+    created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    installation_id BYTEA NOT NULL,
```

Comment:
If we're using raw bytes here it might be better to call it `installation_key` or `installation_public_key` - when shown in UX, we will stringify it and call it `installation_id`: https://github.com/xmtp/libxmtp/blob/main/xmtp_mls/IDENTITY.md#installation-registration

Reply:
Makes sense - rename all the `bytes installation_id` references on the protos to `installation_public_key` too?